src/Hl7.Fhir.Support.Poco/Hl7.Fhir.Poco.Common.csproj

```diff
@@ -0,0 +1,53 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
This file should not be here and should be removed from the solution

Reply:
Mmmmm.....must have been added by VS2019. I did not change this.

Reply:
So, we can remove it, but VS might re-add it when it feels like it....

Reply:
actually, the whole file should be gone ;-)